### DISCUSSION TITLE
Create terminals task does not exist in vega?

### DISCUSSION THREAD

**Discussion by peteruithoven** [requarks/wiki#6226] [Error / Bug Report] [answered]
I've followed the Using VS Code Dev Environment instructions from here:
https://github.com/requarks/wiki/blob/vega/README.md#using-vs-code-dev-environment

I'm in step 6:
> 6. From the Command Palette, run the task "Create terminals":
> - Launch the Command Palette (Ctrl+Shift+P)
> - Type "Run Task" and press Enter
> - Select the task "Create terminals" and press Enter

I run "Tasks: Run Task":
![Screenshot from 2023-03-05 16-26-21@2x](https://user-images.githubusercontent.com/523210/222969730-b9a833be-155c-4868-83d0-f9abbfee68c4.png)

But the "Create terminals" task does not seem to exist:
![Screenshot from 2023-03-05 16-26-02@2x](https://user-images.githubusercontent.com/523210/222969780-60f61774-bda4-4075-95b9-78a896d51cb0.png)

**Comment by NGPixel** [comment, marked as answer]
You're not on the vega branch.

**Reply by peteruithoven** [reply]
You're right, I'm so sorry.
